.circleci/config.yml

```diff
@@ -150,6 +151,54 @@ jobs:
       - run: make image
       - run: make publish-beta-dockerhub
 
+  release_beta_homebrew:
+    docker:
+      - image: artsy/hokusai:beta
+    steps:
+      - add_ssh_keys
+      - run:
+          name: release beta homebrew
+          command: |
+            cd /tmp
+
+            wget https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-beta-Darwin-x86_64.tar.gz
+
+            export UNAME=$(uname)
+            if [ $UNAME == "Darwin" ]; then
+              export SHA256=$(shasum -a 256 hokusai-beta-Darwin-x86_64.tar.gz | awk '{ print $1 }')
+            fi
+
+            if [ $UNAME == "Linux" ]; then
+              export SHA256=$(sha256sum hokusai-beta-Darwin-x86_64.tar.gz | awk '{ print $1 }')
+            fi
+
+            ssh-keyscan github.com >> ~/.ssh/known_hosts
+            git clone git@github.com:artsy/homebrew-formulas.git
+            cd ./homebrew-formulas
+
+            cat <<EOF > ./Formula/hokusai-beta.rb
```

Comment:
`hokusai-beta.rb` does not yet exist, but the first `main` build should create it.